### PR TITLE
Add note in release notes for Volto 17 final

### PR DESCRIPTION
**Pull request by sneridagh**
@ksuess I've figured out what happened: The PR started just before we did the transition to Towncrier, then the change log info was in  `CHANGELOG.md` so it did made to the wrong place, then towncrier didn't pick it up.

Sorry about it!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -55,6 +57,7 @@
 
 ### Feature
 
+- Add control panel for relations @ksuess
```
Comment (davisagli) [inline CHANGELOG.md:60]: You could add a link to https://github.com/plone/volto/pull/3382

**Review by davisagli — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,8 @@
 
 ## 17.0.0 (2023-09-30)
 
+**This is a Release Notes summary of all the changes happened during the alpha stage of Volto 17.**
```
Comment (stevepiercy) [inline CHANGELOG.md:27]:
```suggestion
**These Release Notes summarize all the changes during the alpha stage of Volto 17.**
```

File: CHANGELOG.md (hunk 1 above)
Comment (stevepiercy) [inline CHANGELOG.md:60]:
```suggestion
- Added a control panel for relations. @ksuess [#3382](https://github.com/plone/volto/pull/3382)
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Some gramma and incorporate @davisagli's suggestion.

File: news/5281.documentation [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Add notice in release notes for Volto 17 final to clarify that it's a summary for all the alpha stage @sneridagh
```
Comment (stevepiercy) [inline news/5281.documentation:1]:
```suggestion
Added notice in release notes for Volto 17 final to clarify that it's a summary for all changes during the alpha stage. @sneridagh
```

File: news/5281.documentation [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+Add missing feature for the Relations Control Panel @sneridagh
```
Comment (stevepiercy) [inline news/5281.documentation:2]:
```suggestion
Added an omitted change log entry for the Relations Control Panel. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Pull request merged**